frontend/packages/component-library/src/actionBlock/__tests__/ActionBlock.test.tsx

```diff
@@ -0,0 +1,168 @@
+import {render, screen} from '@testing-library/react';
+
+import '@testing-library/jest-dom';
+import ActionBlock, {FullWidthActionBlock, ActionBlockProps} from '../index';
+
+describe('ActionBlock', () => {
+  const defaultProps: ActionBlockProps = {
+    title: 'Action block title',
+    description: 'This is the action block description.',
+  };
+
+  const primaryButtonProps = {
+    primaryButton: {
+      text: 'Primary Button',
+      href: 'https://code.org',
+      ariaLabel: 'Primary Button aria label',
+    },
+  };
+  const secondaryButtonProps = {
+    secondaryButton: {
+      text: 'Secondary Button',
+      href: 'https://hourofcode.com',
+      ariaLabel: 'Secondary Button aria label',
+    },
+  };
+
+  it('renders the title and description', () => {
+    render(<ActionBlock {...defaultProps} />);
+
+    expect(screen.getByText('Action block title')).toBeInTheDocument();
+    expect(
+      screen.getByText('This is the action block description.'),
+    ).toBeInTheDocument();
+  });
+
+  it('renders an image', () => {
+    render(<ActionBlock {...defaultProps} image="image.png" />);
+
+    expect(screen.getByRole('figure')).toHaveStyle(
+      'background: url(image.jpg) center / cover no-repeat',
```

Comment:
This wouldn't be applicable since we swapped to `<img`?

Reply:
Thanks for catching this!